LICENSE.mustache

```diff
@@ -1,44 +1,105 @@
+## Purpose
 
-If you follow the rules below, you may do everything with this
-software that would otherwise infringe either the contributor's
-copyright in it, any patent claim the contributor can license,
-or both.
+This license lets you use and share this software for
+free, as long as you contribute software you make with it,
+other than prototypes.
 
-1. Contribute changes and additions you make to this software.
+## Acceptance
 
-2. If you combine this software with other software, contribute
-   that other software.
+In order to receive this license, you must agree to its
+rules.  The rules of this license are both obligations
+under that agreement and conditions to your license.
+You must not do anything with this software that triggers
+a rule that you cannot or will not follow.
 
-3. Contribute software you develop, deploy, monitor, or run with
-   this software.
+## Copyright
 
-4. Ensure everyone who gets a copy of this software from you, in
-   source code or any other form, gets the text of this license
-   and the contributor and source code lines above.
+The contributor licenses you to do everything with this
+software that would otherwise infringe their copyright
+in it.
 
-5. Do not make any legal claim against anyone accusing this
-   software, with or without changes, alone or with other
-   software, of infringing any patent claim.
+## Notices
 
-To contribute software, publish all its source code, in the
-preferred form for making changes, through a freely accessible
-distribution system widely used for similar source code, and
-license contributions not already licensed to the public on terms
-as permissive as this license accordingly.
+You must ensure that everyone who gets a copy of any
+part of this software from you, with or without changes,
+also gets the text of this license and the contributor
+and source code lines above.
 
-You are excused for unknowingly breaking 1, 2, or 3 if you
-contribute as required, or stop doing anything requiring this
-license, within 30 days of learning you broke the rule.
+## Patent
 
-**As far as the law allows, this software comes as is, without
-any warranty, and the contributor will not be liable to anyone
-for any damages related to this software or this license, for any
-kind of legal claim.**
+The contributor licenses you to do everything with this
+software that would otherwise infringe any patent claims
+they can license or become able to license.
+
+## Defense
+
+Do not make any legal claim against anyone accusing this
+software, with or without changes, alone or with other
+software, of infringing any patent claim.
+
+## Reliability
+
+The contributor cannot revoke this license.
+
+## Copyleft
+
+With the exception of prototypes:
+
+1.  Contribute all changes and additions you make to
+    this software
+
+2.  If you combine this software with other software,
+    contribute that software.
+
+3.  Contribute software you develop, deploy, monitor,
```

Comment:
It just hit me that `deploy` can be ambiguous here:

1. Deploy *using*: as in, using this software itself as a deploy tool
2. Deploy *with*: as in, including this software in the deployed bundle/binary/etc

Which one is it? Is it both?

Reply:
I'd have thought it was the first on. The other case would be covered by the 'combine' above, no?

Reply:
I actually brought in additional bit of the API Copyleft License that I forgot to call out above:

> 4.  When in doubt, contribute the software.

I would have read "deploy" to cover anything that might be called "deploying". I don't think the word has to have just one meaning in context, any more than we'd have to look at any of the other words in the license, look at their definitions, and pick just one numbered sense of the word.